releasenotes/notes/47957.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+issue: 
+  - 47957
+releaseNotes:
+  - |
+    **Fixed** the command ipilot-agent istio-clean-iptables cannot clean up the iptables rules generated for istio dns proxy.
```

Comment:
```suggestion
    **Fixed** an issue where the `pilot-agent istio-clean-iptables` command cannot clean up the iptables rules generated for istio dns proxy.
```

Reply:
ok, I'll adjust it, thanks for your help.